README.adoc

```diff
@@ -141,7 +134,7 @@ Both packages can be installed from ovirt-master-snapshot-static repository:
   skip_if_unavailable=1
   gpgcheck=0
 
-Please replace `@DIST@` with `fc` for Fedora or `el` for Centos/RHEL.
```

Comment:
I think the repo section can be dropped, we don't use resources.ovirt.org and these packages are available in the copr
```
Both packages can be installed from ovirt-master-snapshot-static repository:
  [ovirt-master-snapshot-static]
  name=Latest oVirt master additional nightly snapshot
  baseurl=http://resources.ovirt.org/pub/ovirt-master-snapshot-static/rpm/@DIST@$releasever/
  enabled=1
  skip_if_unavailable=1
  gpgcheck=0
  
  Please replace `@DIST@` with `fc` for Fedora or `el` for Centos/RHEL.
  ```

Reply:
@mwperina please dis/approve

Reply:
Yes, it should be dropped, ovirt-master-release is only on COPR: https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/